js/src/options.ts

```diff
@@ -20,6 +20,7 @@ export type Options = {
     onPreferencesChanged?: (value: string) => any;  // Executes when the user's preferences have changed.
     disableGrammar?: boolean;       // Option to disable Grammar
     disableTranslation?: boolean;   // Option to disable translation
+    disableLanguageDetection?: boolean;      // Option to disable language detection
```

Comment:
maybe add that comment piece from the Kaiju PR?

// Option to disable language detection and use only the language that is explicitly specified on the chunk

Reply:
done